src/pybind/mgr/nfs/export.py

```diff
@@ -325,6 +325,17 @@ def _create_export_user(self, export: Export) -> None:
             rgwfsal.secret_access_key = j['keys'][0]['secret_key']
             log.debug("Successfully fetched user %s for RGW path %s", rgwfsal.user_id, export.path)
 
+    def _ensure_cephfs_export_user(self, export: Export) -> None:
+        assert isinstance(export.fsal, CephFSFSAL)
+        fsal = cast(CephFSFSAL, export.fsal)
```

Comment:
i do not understand this, it is first asserted that `export.fsal` be of type `CephFSFAL` and then it is casted to type `CephFSFSAL` only if it is of type `CephFSFSAL`?

Reply:
`fsal = cast(CephFSFSAL, export.fsal)` should be enough!  will fix it, thanks for noticing!

Reply:
This looks like is pending to be cleaned up